README.md

```diff
@@ -38,6 +38,7 @@ This table represents the supported components of AWS OTel Collector in 2020. Th
 |                                 |                               | newrelicexporter                   |                        |
 |                                 |                               | sapmexporter                       |                        |
 |                                 |                               | signalfxexporter                   |                        |
+|                                 |                               | loadbalancingexporter              |                        |
```

Comment:
We need to go thru AWS AppSec for any new components will be added into ADOT.

Reply:
OK. I will split unit test fixing and create another PR to fix unit test in go1.16.

Reply:
Thanks, do you think we can merge the new component support to `prometheus` branch until it's ready for the release?

Reply:
Yes. I have changed this PR to `premetheus` branch.